pages/blog/2023-summary.md

```diff
@@ -0,0 +1,169 @@
+
+If you are curious about our past achievements, [please read the previous summary](https://www.asyncapi.com/blog/2022-summary).
+
+## Community Metrics
+What better way to reflect on the past year than with numbers? This is the most significant way to gauge our growth and measure our visible impact as an open-source community.
+
+### Slack
+As of 2024, we are currently sitting at an overall total of **4,471** members in the AsyncAPI workspace. This is where all our community activities happen, from communication to sharing updates and helping contributors and new adopters for the AsyncAPI spec and tools. 
+
+In 2023, we had 1488 people joining our workspace:
+
+<Figure
+  src="/img/posts/2023-summary/new-users-joining-per-year.webp"
+/>
+
+Below are our top 5 members who joined the community in 2023 and have been the most active among other newcomers. As a community, we are proud of their contributions as part of our Mentorship programs and even keeping us entertained with memes 😀.
+
+<Figure
+  src="/img/posts/2023-summary/Top-5-members.webp"
+/>
+
+
+Additionally, our weekly activity continues to grow as we run different mentorship programs and initiatives that bring traction to our community.
+
+<Figure
+  src="/img/posts/2023-summary/slack-activity.webp"
+/>
+
+
+### Social Media 
+
+Social media is a medium we use for sharing news and updates around our community. We don't use it for marketing purposes but mainly for communication.
+
+
+> And honestly, I didn't invest much when it came to socials, so don't expect big big numbers 😝
+
+Let's dive into the data:
+
+#### LinkedIn
+
+For our LinkedIn page, we are currently sitting at **2988** overall followers, and during 2023, we gained a total of **849** followers since 2022.
+
+<Figure
+  src="/img/posts/2023-summary/linkedin.webp"
+/>
+
+We also don't use any premium or marketing services; everything you see here is 100% organic, and even though LinkedIn still has much to offer users, it is way better than the X/Twitter app, in my opinion.
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-LinkedIn.webp"
+/>
+
+#### X/Twitter
+
+I don't have much to say regarding Twitter, but on the bright side, we gained almost 1K followers since 2022, and we currently have 4.4k overall followers. When it comes to website referrals via Twitter, we have seen a decline compared to last year, 2022
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-Twitter.webp"
+/> 
+
+Although the decline is shallow, I'm not extremely worried. Most top tweets for this year were appreciation tweets balanced up with technical tweets. Additionally, we don't use any paid services, and having any additional investment in this platform is not worth it, at least for now.
+
+#### YouTube
```

Comment:
super nice to see we are not loosing much because we moved from online to in-person

can you also include info on how many subscribers we have and how many joined this year?